client/src/main/java/com/kritsit/casetracker/shared/domain/model/Evidence.java

```diff
@@ -10,7 +10,7 @@
     private String description;
     private File serverFile;
     private File localFile;
-    private BufferedImage image;
+    private transient BufferedImage image;
```

Comment:
"Non-transient non-serializable instance field in serializable class"

BufferedImage is not serializable, at least in the typical way. I transient marks it not to be serialized. How are you storing these images?